chore: add CONTRIBUTING.md (#307)
nuskey8/Lua-CSharp: CONTRIBUTING.md

# Contributing

## Dependencies

> The following are used in the development of Lua-CSharp:

Reviewer: ```suggestion
The following tools are used for Lua-CSharp development:
```
sounds more natural
